### DISCUSSION TITLE
Add an installation instructions to readme/installation.md

### DISCUSSION THREAD

**Discussion by taep96** [ppy/osu#15843] [Ideas]
I think it would be a good idea to provide official installation instructions in README or a separate file(to keep README short) and link it 🤔

**Comment by bdach** [comment]
windows has a bundled installer. other platforms have binaries that are immediately installable/executable. additionally a better installer is planned (https://github.com/ppy/osu/issues/15731). the installer should be the ultimate "installation guide".

nobody reads readmes. you can trust me by the number of times i've had to quote the existing contributing guidelines at people.

**Reply by taep96** [reply]
I played Lazer on windows and Linux and can agree that Windows installer is enough, but AppImage needs to be set up for it to work properly. By properly I mean it needs to show up in app launcher with icon and launch from there. I doubt that people will be satisfied with running a file that looks like this: 
![image](https://user-images.githubusercontent.com/64481039/143719442-f4175d04-cd48-4cee-a69f-fff22545f3be.png)
from their desktop or a folder and not being able to find search for it or pin it on the dock.

Alternatively, there could be an install script that downloads latest release and does all the setup, but the setup is probably different on other Linux distributions.

I already have an idea on instructions for Ubuntu and Ubuntu based distributions. I can make a draft if you want me to.

**Reply by bdach** [reply]
linux users are expected to be able to figure it out. they're running linux after all. improving the experience there is definitely a work item to be addressed but it's not a high priority.

also the game has already been unofficially added to several distro package managers (AUR, flathub). more may come at a later stage when the time is right.

**Reply by taep96** [reply]
> linux users are expected to be able to figure it out. they're running linux after all.

Sure, in the state that Linux is in now it's not very newbie-friendly and mostly advanced users use it, but same goes for Windows. I consider it unfair to leave out an OS just because it's less popular. Figuring out could be left to users of less known distros, since they already chose the hard way and probably won't have problem with that.

**Reply by taep96** [reply]
If I make a repository with installation instructions and (later) scripts for most major distributions. Is there a chance that it could be linked somewhere officially?

**Reply by bdach** [reply]
> I consider it unfair to leave out an OS just because it's less popular

again, this is a *transitional* state of affairs. stable was officially available for windows only. the primary goal on the horizon is to let those users migrate over first. *all other* platforms are considered secondary in the mean time, including mobile.

> If I make a repository with installation instructions and (later) scripts for most major distributions. Is there a chance that it could be linked somewhere officially?

if the instructions boil down to `$ chmod +x osu.AppImage` then probably not. to reiterate, a proper OS-native installation path is preferable to some linked document.

**Reply by taep96** [reply]
>if the instructions boil down to `$ chmod +x osu.AppImage` then probably not

Since that is not going to be the case I will do it, and send the link as an answer later.

>again, this is a transitional state of affairs. stable was officially available for windows only. the primary goal on the horizon is to let those users migrate over first. all other platforms are considered secondary in the mean time, including mobile.

Should, have considered that, thanks for explaining.

**Reply by peppy** [reply]
If you can't handle an executable file, go use an unofficial package on your package manager.
